src/models/decomposition_models.jl

```diff
@@ -43,7 +43,7 @@ function MMI.fit(model::PCA, verbosity::Int, X)
     )
     cache = nothing
     report = (
-        indim=MS.size(fitresult,1),
+        indim=MS.size(fitresult,1)
```

Comment:
you are right after all about some issue here, but fixed in https://github.com/JuliaAI/MLJMultivariateStatsInterface.jl/pull/47

Reply:
I think that to avoid merge conflict, you can just put the common back:

```suggestion
        indim=MS.size(fitresult,1),
```